### PR TITLE
added a blog

### PR DESCRIPTION
**Pull request by sidny**
post a blog for Support-Developers 

wxid: sidny00

### CODE REVIEW

File: _posts/2020-07-12-wechat-roommessage-bot.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+---
+title: "群消息同步机器人"
+author: [@sidny](https://github.com/sidny) 
+date: "2020-07-12 15:00:00 +0800"
+categories: project
+tags:
+  - wechaty
+  - wechaty-puppet-padplus
```
Comment (lijiarui) [inline _posts/2020-07-12-wechat-roommessage-bot.md:8]: Please add a teaser for this blog.

**Review by lijiarui — CHANGES_REQUESTED** [review]

File: _posts/2020-07-12-wechat-roommessage-bot.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+---
+
+<!-- markdownlint-disable -->
```
Comment (lijiarui) [inline _posts/2020-07-12-wechat-roommessage-bot.md]: Please remove markdown link disable.

**Review by lijiarui — CHANGES_REQUESTED** [review]

File: _posts/2020-07-12-wechat-roommessage-bot.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
+---
+title: "群消息同步机器人"
+author: [@sidny](https://github.com/sidny) 
+date: "2020-07-12 15:00:00 +0800"
+categories: project
+tags:
+  - wechaty
+  - wechaty-puppet-padplus
+header:
+  teaser: /assets/2020/wechat-roommessage-bot/teaser.png
```
Comment (sidny) [inline _posts/2020-07-12-wechat-roommessage-bot.md:10]: add teaser

File: _posts/2020-07-12-wechat-roommessage-bot.md (hunk 1 above)
Comment (sidny) [reply _posts/2020-07-12-wechat-roommessage-bot.md:8]: done

**Review by sidny — COMMENTED** [review]
1 remove markdown link disable   --- done 
2 add teaser --- done

File: _posts/2020-07-12-wechat-roommessage-bot.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+- 回复
+  ![回复](/assets/2020/wechat-roommessage-bot/answer.png)
+
+- 公告
+  ![公告](/assets/2020/wechat-roommessage-bot/notice.png)
+
+### 致谢
+
+非常感谢Wechaty团队提供微信机器人SDK，让开发者可以专注于业务代码。
+感谢句子互动提供的pad协议版token。
```
Comment (lijiarui) [inline _posts/2020-07-12-wechat-roommessage-bot.md]: Please add link for wechaty and 句子互动：

- wechaty： https://wechaty.github.io/
- 句子互动： https://www.juzibot.com/

**Review by lijiarui — CHANGES_REQUESTED** [review]
Good job, please add link for the blog and then I will merge this PR.

**Review by lijiarui — APPROVED** [review]
Good job！

**Comment by lijiarui** [comment]
Congratulations! 🎉 🎉 🎉

You become a wechaty contributor successfully! 👏 👏 👏

Please:

- [ ]  Add my wechat: ruirui_0914 and send me this pr link. I will invite you into Wechaty Contributor Room
- [ ] Update your profile here: [Contributor List](https://docs.google.com/spreadsheets/d/1XcDoIczyIclqXP1p90Sz7S0n4Q22xjFzJmjPFlU2g1E/edit#gid=0), then other contributors will know you better!
- [ ]  Join wechaty org by clicking https://github.com/wechaty

image

**Comment by sidny** [comment]
Add my wechat: ruirui_0914 and send me this pr link   **--- done**
 Update your profile here: Contributor List   **---- done**
 Join wechaty org by clicking https://github.com/wechaty    **---- done**

**Pull request merged**